runtimes/common/src/constants.rs

```diff
@@ -194,11 +204,11 @@ pub mod governance {
 }
 
 pub mod did {
-	use crate::BlockNumber;
+	use super::*;
 
-	use super::{Balance, HOURS, KILT, MILLI_KILT};
+	pub const DID_SIZE: u32 = 7418;
 
-	pub const DID_DEPOSIT: Balance = 2 * KILT;
+	pub const DID_DEPOSIT: Balance = deposit(2 + MAX_NUMBER_OF_SERVICES_PER_DID, DID_SIZE);
```

Comment:
Where does the 2 come from? Is that details + endpoints count entries?

Reply:
yes